applications/rag/README.md

```diff
@@ -96,32 +98,38 @@ curl 127.0.0.1:8080/generate -X POST \
+This step generates the vector embeddings for your input dataset. Currently, the default dataset is [Google Maps Restaurant Reviews](https://www.kaggle.com/datasets/denizbilginn/google-maps-restaurant-reviews). We will use a Jupyter notebook to run a Ray job that generates the embeddings & populates them into the instance `pgvector-instance` created above.
 
-1. Download the provided Juypter notebook to generate vector embeddings from `ai-on-gke\applications\rag\example_notebooks\ray-hf-cloudsql-latest.ipynb`.
+1. Create a CloudSQL user to access the database: `gcloud sql users create rag-user-notebook --password=<choose a password> --instance=pgvector-instance --host=%`
 
-2. Create a CloudSQL user to access the database: `gcloud sql users create rag-user-notebook --password=${PASSWORD:?} --instance=pgvector-instance --host=%`
+2. Go to the Jupyterhub service endpoint in a browser: `kubectl get services proxy-public -n <namespace> --output jsonpath='{.status.loadBalancer.ingress[0].ip}'`
 
-3. Go to the Jupyterhub service endpoint in a browser: `kubectl get services proxy-public -n <namespace> --output jsonpath='{.status.loadBalancer.ingress[0].ip}'`
-
-4. Login with placeholder credentials [TBD: replace with instructions for IAP]:
+3. Login with placeholder credentials [TBD: replace with instructions for IAP]:
 * username: user3
 * password: use `terraform output password` to fetch the password value
 
-5. Once logged in, choose the `CPU` preset & use the Upload button to upload the notebook `ray-hf-cloudsql-latest.ipynb`. Replace the variables in the 3rd cell with the following:
+4. Once logged in, choose the `CPU` preset. Go to File -> Open From URL & upload the notebook `rag-kaggle-ray-sql.ipynb` from `https://raw.githubusercontent.com/GoogleCloudPlatform/ai-on-gke/main/applications/rag/example_notebooks/rag-kaggle-ray-sql-latest.ipynb`. This path can also be found by going to the [notebook location](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/example_notebooks/rag-kaggle-ray-sql-latest.ipynb) and selecting `Raw`.
 
+5. Replace the variables in the 3rd cell with the following to access the database:
 * `INSTANCE_CONNECTION_NAME`: `<project_id>:<region>:pgvector-instance`
 * `DB_USER`: `rag-user-notebook`
-* `DB_PASS`: password from step 2
+* `DB_PASS`: password from step 1
+
+6. Create a Kaggle account and navigate to https://www.kaggle.com/settings/account and generate an API token. See https://www.kaggle.com/docs/api#authentication how to create one from https://kaggle.com/settings ([screenshot](https://screenshot.googleplex.com/4rj6Tjdwt5KGTRz)). This token is used in the notebook to access the [Google Maps Restaurant Reviews dataset](https://www.kaggle.com/datasets/denizbilginn/google-maps-restaurant-reviews)
```

Comment:
https://screenshot.googleplex.com/4rj6Tjdwt5KGTRz needs to be replaced with actual snapshot or remove this link